README.md

```diff
@@ -44,9 +44,20 @@ var benchmarksSchema = qppMeasuresData.getBenchmarksSchema();
 
 ### Generating measures-data.json
 To regenerate the `measures-data.json` file, which contains additional metadata and conforms to
-the measures schema, run:
+the measures schema, do the following:
+
+1. download the qpp quality pdfs 
+	```
+	wget https://qpp.cms.gov/docs/QPP_quality_measure_specifications.zip .
+	unzip QPP_quality_measure_specifications.zip
+	unzip Claims-Registry-Measures.zip
+	```
+2. run the convert from pdfs tool to get the quality info from the pdfs
+	```
+	node scripts/get-quality-measures-from-pdfs.js Claims-Registry-Measures
```

Comment:
Hm, I'm having trouble recreating this. Is the Claims-Registry-Measures folder unzipped in the path that you ran this from?